src-docs/src/views/timeline/timeline_example.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { GuideSectionTypes } from '../../components';
+
+import {
+  EuiCode,
+  EuiTimeline,
+  EuiTimelineItem,
+} from '../../../../src/components';
+import timelineItemConfig from './playground';
+
+import Timeline from './timeline';
+const timelineSource = require('!!raw-loader!./timeline');
+const timelineSnippet = `<EuiTimeline
+  timelines={[
```

Comment:
Whoops, found one typo
```suggestion
  items={[
```